docs/contributing/software-req-win.md

```diff
@@ -142,14 +142,14 @@ Run the following from a Windows PowerShell prompt (elevation is not required):
 To run unit tests, ensure you have created the nativebuildimage above.
 Then run one of the following from an (elevated) Windows PowerShell prompt:
 
-    docker run --rm nativebuildimage hack\make.ps1 -TestUnit
-    docker run --rm -m 2GB nativebuildimage hack\make.ps1 -TestUnit  # (if using Hyper-V containers)
+    docker run --rm -e DOCKER_GITCOMMIT=$DOCKER_GITCOMMIT nativebuildimage hack\make.ps1 -TestUnit
+    docker run --rm -m 2GB -e DOCKER_GITCOMMIT=$DOCKER_GITCOMMIT nativebuildimage hack\make.ps1 -TestUnit  # (if using Hyper-V containers)
```

Comment:
Given that the environment variable matches the one from the environment, this could be

```suggestion
    docker run --rm -e DOCKER_GITCOMMIT nativebuildimage hack\make.ps1 -TestUnit
    docker run --rm -m 2GB -e DOCKER_GITCOMMIT nativebuildimage hack\make.
```

(when passing `-e SOME_ENV_VAR` without a value (no `=`), the CLI will lookup the given env-var, and (if set) set the value to the value set for that env-var)


I'm curious though why we make the script fail in this case; or more specifically;

- why we make it fail when running _unit tests_ (which should not require the git-commit)
- if (in general) this should just be a warning; when building the binaries, we want the git-commit to be set to some value (as it's included in `docker version` and `dockerd --version`), but I _think_ it doesn't have to be more than a _warning_ (we can set a default `<unknown>` or empty value, or ...

Reply:
cc @crazy-max @vvoland